package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "msw-client",
-  "version": "0.0.96",
+  "version": "0.0.87",
```

Comment:
Hey Dorit.
I see there's a drop in version. 
This can create problems later in master.